lib/bucky/core/test_core/test_manager.rb

```diff
@@ -45,8 +89,16 @@ def load_test_suites
 
         # Generate and execute test
         def do_test_suites(test_suite_data)
-          parallel_num = Bucky::Utils::Config.instance[:parallel_num]
-          parallel_helper(test_suite_data, parallel_num)
+          # For checking on linkstatus
+          link_status_url_log = {}
```

Comment:
I think unnecessary variables are defined.
When test category is e2e, variables for linkstatus are unnecessary.
How about you?

Reply:
I'll fix it!